Nosenti/certificate-manager---Innocent
# Task1-KAN-4 setup the folder structure

#### PR description
- This PR is for setting up the project: folder structure, ESLint and Webpack

#### Ticket / Jira Link
- [KAN-4](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-4)

#### Checklist:

#### How To Test Changes
- Clone the repo
- Inside the repo, run npm start
- In the browser, go to localhost:3000 and project should run

#### Screenshots:
![image](https://github.com/Nosenti/certificate-manager---Innocent/assets/36986840/e05fa58b-2daa-430b-a8d4-8d0956d86d64)
